.github/pull_request_template.md

```diff
@@ -10,27 +11,28 @@ https://github.com/publicsuffix/list/wiki/Guidelines#submit-the-change
 
 Also, read them again, as many skip that part and 
 get confused about why their PR is delayed or does
-not get accepted when theirs didn't follow them.
+not get accepted when their submission didn't follow them.
 
 A recent PR using the current template is 
 https://github.com/publicsuffix/list/pull/1591, although 
 the organization and description were not as substantial 
 as desired, which required maintainers time to visit the 
-requestors website to further research. 
-Having more robust org/desc improves the PR processing 
-pace due to the extra cycles not lost to research.
+requestors website to further research.
+Having more robust responses improves the PR processing 
```

Comment:
We literally want more robust responses to both the organization and the description, which was the origin of the specific wording.  Absent explaining this we were getting really basic crap answers or cut-paste marketing smeg that did not articulate the rationale for inclusion in PSL

Reply:
so maybe, please, leave this as ```org/desc```

Reply:
https://github.com/publicsuffix/list/pull/2156/commits/bb05a8e667f631f4a41d3837285903fe57eec6a4